boards/arm/thingy53_nrf5340/thingy53_nrf5340_cpunet_defconfig

```diff
@@ -13,5 +13,9 @@ CONFIG_HW_STACK_PROTECTION=y
 # Enable GPIO
 CONFIG_GPIO=y
 
+# Enable uart driver
+CONFIG_SERIAL=y
+
```

Comment:
Could you please place this below `CONFIG_UART_CONSOLE=y`? This is just to be aligned with app core. I found those little things helpful when was adding the board files and those look "more or less the same".

Reply:
Forget it, its already a mess.

Reply:
I made it in the same way as it was for nRF53 DK

Reply:
Yes, i know, forget it, sorry.